popUp.js

```diff
@@ -0,0 +1,98 @@
+const body = document.querySelector('body');
+const overlay = document.getElementById('overlay')
+const workData = [{
+    name: 'Keeping track of hundreds of components',
+    description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Itaque aperiam, laudantium repellat maiores rerum id recusandae ratione error, fuga modi sapiente sequi minima vitae similique voluptas perferendis, eum possimus aut.',
+    technologies: ['Codekit', 'Github', 'Javascript', 'Bootsrap', 'Terminal', 'Codepen'],
+    liveLink: 'https://khitermedachraf.github.io/my-professional-portfolio/',
+    source: 'https://github.com/khitermedachraf/my-professional-portfolio',
```

Comment:
- Good job creating an object for the popup in the JavaScript file 👍 There are a few issues to work on concerning the project cards.
- [ ] Kindly remove all the code of the project cards from your HTML file because the project cards are to be rendered dynamically from JavaScript and not coded in the HTML file.
- [ ] Kindly create an array of objects in the JavaScript file to store JavaScript objects containing the information for each project. Each object should contain at least the following information about the project: name, description, featured image, technologies, and link to the project. So you should have something like this:
- Array of JavaScript objects
![Annotation 2022-04-06 111124](https://user-images.githubusercontent.com/85113567/161952686-f3c59f89-84ea-46ae-9bd0-9842173f5472.jpg)

- [ ] Now loop through the array of the project objects and render them dynamically from JavaScript. This will make it easier to add a new project in the future since you would just need to add that project's information into the array 💯

Reply:
I got it, Sir, Thank you so much for the professional review and the pieces of advice, I will  work on the changes.

Reply:
- [ x ] I am done with all the required changes.